vmdb/spec/models/ems_refresh/refreshers/kubernetes_refresher_spec.rb

```diff
@@ -127,4 +129,14 @@ def assert_specific_container_service
       :namespace      => "default"
     )
   end
+
+  def assert_specific_container_replicator
+    @replicator = ContainerReplicator.find_by_name("monitoring-influx-grafana-controller")
```

Comment:
Rails 4.2 is on the horizon, so let's use arel format as some find_by methods are deprecated in rails 4:
http://blog.remarkablelabs.com/2012/12/what-s-new-in-active-record-rails-4-countdown-to-2013

```ruby
@replicator = ContainerReplicator.where(:name => "monitoring-influx-grafana-controller").first
```